examples/advanced/stereo_matching/main.py

```diff
@@ -42,6 +42,7 @@
 bproc.material.add_alpha_channel_to_textures(blurry_edges=True)
 bproc.renderer.toggle_stereo(True)
 bproc.renderer.set_max_amount_of_samples(50)
+bproc.renderer.enable_segmentation_output(map_by=["category_id", "instance", "name"])
```

Comment:
Why do we add this here? Because it works now? :D

Reply:
So that it doesn't break again in future? :D I heard it's good practice to add tests to fixes. And we might refactor some of the code one day.

Reply:
What a compelling argument :D